docs/modules/ROOT/pages/index.adoc

```diff
@@ -323,13 +325,36 @@ The `signer_payload` data structure MUST be presented to be signed by the _<<_cr
 
 IMPORTANT: If a data hash assertion is being used, the _<<C2PA claim generator>>_ MUST also follow the process described in xref:_interaction_with_data_hash_assertion[xrefstyle=full].
 
+==== Calculating the `expected_claim` value
+
+If the *<<_identity_assertion,identity assertion>> generator* chooses to include the `expected_claim` value in `signer_payload`, it MUST be computed as follows:
+
+. Create a valid _<<C2PA claim>>_ data structure that includes completed references to all _<<_c2pa_assertion,C2PA assertions>>,_ *including* all *<<_identity_assertion,identity assertion>>.*
+. For the specific *<<_identity_assertion,identity assertion>>* being computed, replace the `hash` value with 32 zero-value (`0x00`) bytes.
+. Serialize the _<<C2PA claim>>_ data structure using link:++https://www.rfc-editor.org/rfc/rfc8949.html#name-core-deterministic-encoding++[Section 4.2.1, “Core deterministic encoding,”] of RFC 8949: Concise binary object representation.
+. Compute a SHA-256 hash of the serialized _<<C2PA claim>>_ data structure.
+
+The resulting hash should be used as the `expected_claim` value.
+
+IMPORTANT: Since this algorithm replaces the `hash` value for exactly one *<<_identity_assertion,identity assertion>>,* this implies that only one *<<_identity_assertion,identity assertion>>* per _<<C2PA Manifest>>_ may provide an `expected_claim` value and that this *<<_identity_assertion,identity assertion>>* MAY NOT be referenced by any other *<<_identity_assertion,identity assertion>>.*
```

Comment:
Just thinking about the case of more than one identity assertion being present. 

At first glance it looks like if the claim hash for each identity assertion were zeroed out when calculating the expected claim value for each identity assertion signature payload, then it is possible to include more than one. However, if this were the case then the contents of identity assertion 1 say could be updated after the claim was published in collusion with the c2pa claim manifest signer (or even another signer)  unbeknown to the other identity assertion providers. This is because identity assertion 1 payload was not represented in identity assertion 2's expected claim value.

A solution to this might be that when calculating the expected claim value for an identity assertion, then.. 

1. When considering the hashed uri value for itself it does as suggested  and zeroes out. This is fine since it is signing its own referenced assertion payload anyway and the claim generator will subsequently sign the whole identity assertion with the blob hash as usual
2. When considering the hashed uri values of other identity assertions, it uses the hash of that target identity referenced assertions payload. This is known at the time of signing and can be verified at validation. Note that the validator that cannot parse an identity assertion is happy that the whole box hash is correct in the claim box and doesn't care about its contents whereas validators that parse the identity assertion can reconstitute each of the reference payload hashes.
3. To make this easier we could include 'referenced assertions hash' as a field in the signature payload, much like xmlsig includes the hash of the payload as an explicit tbs field